lib/functional.js

```diff
@@ -4,4 +4,30 @@ module.exports = (api) => {
 
   api.common.curry = (fn, ...args) => fn.bind(null, ...args);
 
+  api.common.zip = (...arrs) => {
```

Comment:
Please add short description in sources and we need tests which can be used as examples